bliss/surveys/sdss.py

```diff
@@ -92,6 +100,30 @@ def __getitem__(self, idx):
             self.items[idx] = self.get_from_disk(idx)
         return self.items[idx]
 
+    def image_id(self, idx) -> Tuple[int, int, int]:
+        """Return the image_id for the given index."""
+        return self.rcfgcs[idx][0], self.rcfgcs[idx][1], self.rcfgcs[idx][2]
```

Comment:
`self.rcfgcs` is already a tuple, right? Is there a reason for indexing it separately instead of just `self.rcfgc[idx][0:3]`?